sap/5.5/modules/ROOT/pages/sap-connector-examples.adoc

```diff
@@ -690,6 +698,499 @@ output application/java
 ----
 
+
+[[manage-transactions]]
+== Manage Transactions
+
+This example performs two calls on standard BAPIs that handle bank records and manages those transactions.
+
+The following screenshots show the Studio flows for this example:
+
+image::managetransactions-flow1.png[Studio flow for the first flow of Managing Transactions]
+
+image::managetransactions-flow2.png[Studio flow for the second flow of Managing Transactions]
+
+image::managetransactions-flow3.png[Studio flow for the third flow of Managing Transactions]
+
+
+=== Configure the First Flow
+
+The first flow retrieves a list of available banks in the United States, filters the first result, and fetches information about the bank. To configure this flow:
+
+. In Anypoint Studio, create a new Mule project.
+. From the *Mule Palette* view, select *HTTP* and drag the *Listener* operation to the canvas to start a new flow.
+. Configure the global element using the default values.
+. In the *Listener* properties tab, set the path to `/sync-rfc`.
+. Drag a *Transform Message* component next to *Listener*.
++
+The content of this message filters the banks by country.
+. In the *Output* section of the *Transform Message* component, overlay the brackets with this text:
++
+----
+%dw 2.0
+output application/xml
+---
+{
+	BAPI_BANK_GETLIST: {
+		"import": {
+			BANK_CTRY: "US",
+			MAX_ROWS: 0
+		}
+	}
+}
+----
+
+. From the *Mule Palette* view, select *SAP* and drag the *Synchronous Remote Function Call* operation next to the *Transform Message* component.
+. Create a global element named `SAP_Config` and specify the connection information.
+. Click *Test Connection* to confirm that Mule runtime engine can connect with the SAP instance.
+. Enter `BAPI_BANK_GETLIST` for the *Function Name* and `payload` for the *Content*.
+. Drag another *Transform Message* component next to *Synchronous Remote Function Call*.
++
+The content of this message filters the first result.
+. In the *Output* section of the *Transform Message* component, overlay the brackets with this text:
++
+----
+%dw 2.0
+output application/xml
+---
+{
+	BAPI_BANK_GETDETAIL: {
+		"import" : {
+			BANKCOUNTRY: payload.BAPI_BANK_GETLIST.tables.BANK_LIST.*row[0].BANK_CTRY,
+			BANKKEY: payload.BAPI_BANK_GETLIST.tables.BANK_LIST.*row[0].BANK_KEY
+		}
+	}
+}
+----
+. From the *Mule Palette* view, select *SAP* and drag another *Synchronous Remote Function Call* operation next to the *Transform Message* component.
+. Create a global element named `SAP_Config` and specify the connection information.
+. Click *Test Connection* to confirm that Mule runtime engine can connect with the SAP instance.
+. Enter `"BAPI_BANK_GETDETAIL"` for the *Function Name* and `payload` for the *Content*.
+. In the *Mule Palette* view, select *Core* and drag a *Logger* component next to *Synchronous Remote Function Call*.
+
+=== Configure the Second Flow
+
+The second flow creates a sales order from the data. To configure this flow:

```

Comment:
Here we are adding a Try around the BAPI call to create a transaction, if not, the change does not impact on SAP.